src/components/Pagination/index.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react'
+import { PaginationLeftItem, PaginationRightItem, PaginationPanel } from './styled'
+import LeftBlack from '../../assets/pagination-black-left.png'
```

Comment:
Why use `-` for  image name ? Please make sure to be consistent with other image name.